packages/server/src/middleware/modern.js

```diff
@@ -34,19 +34,28 @@ const detectModernBuild = ({ options, resources }) => {
   }
 }
 
-const detectModernBrowser = (req, options) => {
+const detectModernBrowser = (req) => {
+  const { socket = {}, headers } = req
```

Comment:
Destructuring can be in function params :)

Reply:
OK, I’ll change it shortly